### ISSUE TITLE
Funding triage / schedule

### ISSUE THREAD

**Issue by lrettig** [gitcoinco/ERC-1789#21]
It will take some time to figure out the right DAO design (#5). In the meantime, I propose we come up with a MVP design that's as simple, non-controversial, and trust-minimized as possible. Here's my proposal.

## General principles

* All funded projects must commit to open source (which license?) and transparency of use of funds
* No on-the-ground legal entity or bank account; no fiat; all funds disbursed on chain
* Technical steering committee (#22) (seven or nine seats) to manage funds/make funding decisions.
* Ideally have stewardship of no funds. Where that’s impossible, keep funds as briefly as possible. Pre-commit to use of funds before they’re received. Make credible commitments in code wherever possible.
* Roadmap to full decentralization/DAOification/trust minimization.
* Publish best practice on budget, calculating fees, etc. and work with teams to get this right
* Which values/principles/criteria will be used when deciding how to allocate funds?

## Phase 0: Commitment

Timing: Istanbul hardfork, Q3/4 2019
Funding target: No rewards collected.

In parallel, continue to work on other avenues of funding/alternate structures e.g. Moloch, donations from large projects. Continue to work on DAO design and governance—set up steering committee.

## Phase 1: Bootstrapping

Timeframe: Istanbul+1, Q1/Q2 2020
Funding target: $250k p.a.

In the initial phase we optimize for simplicity and non-controversiality. We only seek to provide funding to the two most important mainnet client teams: Geth and Parity. Without these teams and these projects, Ethereum would immediately cease to function.

We interview the leads of both of these teams to understand the project roadmaps, funding requirements, and current amount of funding. We work with the teams to draw up transparent budgets, which IMHO are a requirement for this to work. (Note: We don't necessarily need transparent granularity down to, e.g., the individual fee/salary level. We can aggregate things to some extent to preserve privacy.)

Collect 10-20% of total funding to begin funding governance research.

Keep working with ecosystem projects to understand and map out funding requirements for later phases.

## Phase 2: Layer one

Timeframe: Late 2020
Funding target: $1-3M p.a.

At this point the basic model has been proven out, and we should have established trust with the community. Provide funding for 3-4 mainnet clients, and other layer one projects such as Swarm, EVM evolution, and Ewasm, and possibly other critical ecosystem work such as project management.

Fund one full-time governance researcher.

## Phase 3: Layer two

Timeframe: Early 2021
Funding target: $5-10M p.a.

Offer funds to critical layer two protocols.

## Phase 4: Grants

Timeframe: Late 2021
Funding target: $10-20M p.a.

At this point we can begin to accept funding applications from projects at layers one and two.

## Phase 5: DAOification

Timeframe: 2022 and beyond
Funding target: $20M+ p.a.

**Comment by owocki** [comment]
> Collect 10-20% of total funding

This means "Collect 10-20% of total *inflation* funding"? 

One idea I'd propose to add to the "Commitment" phase is, perhaps we can raise some limited amount of funds to cover administrative costs and also bootstrap the funding decision making process.  Perhaps we could give out a few small test grants just to prove to the community how the steering committee would operate.

**Comment by lrettig** [comment]
> This means "Collect 10-20% of total inflation funding"?

Yup

> One idea I'd propose to add to the "Commitment" phase is, perhaps we can raise some limited amount of funds to cover administrative costs and also bootstrap the funding decision making process

How would you raise these funds? Gitcoin?

**Comment by owocki** [comment]
Yes, I'm going to try to campaign for this via Gitcoin Grants in [CLR Round 2](https://medium.com/gitcoin/gitcoin-grants-50k-open-source-fund-e20e09dc2110).  The more of a groundswell of contributions we can get the more the matching will be.

**Comment by GregTheGreek** [comment]
My comments from a prior telegram chat:

> The dates imo should be the deadlines. But speeding things up where possible (things change) is best. Otherwise structure is great.

**Comment by jpitts** [comment]
These phases being themed w/ "Layer 1", "Layer 2", "Grants" seems arbitrary. Example: Is education that much more difficult to fund than Layer 1 initiative or hiring a governance researcher? 

Why not first establish the priorities (or a process to establish the priorities), and then use these phases to step up the amount raised / number of projects funded?

**Comment by lrettig** [comment]
> Is education that much more difficult to fund than Layer 1 initiative or hiring a governance researcher?

It's not about difficulty, it's about degree of controversiality. One could make the case that education is not worth investing in, or that it should not be viewed as a public good. It's much harder IMHO to make that case about something like go-ethereum which is the cornerstone of the entire network today, is open source, etc.

**Comment by jpitts** [comment]
It may be prevalent view, but I see that as disrespectful to the work of contributors. Still, I get it, we aren't even able to do anything without the layers below: 

protocol researchers -> layer 1 devs -> dapp devs and network operators -> uxers -> community folk and educators

But perhaps we can create easing in, allocate more toward layer 1 initially, but create coverage for all according to priority.

**Comment by pet3r-pan** [comment]
> It may be prevalent view, but I see that as disrespectful to the work of contributors. Still, I get it, we aren't even able to do anything without the layers below:
> 
> protocol researchers -> layer 1 devs -> dapp devs and network operators -> uxers -> community folk and educators
> 
> But perhaps we can create easing in, allocate more toward layer 1 initially, but create coverage for all according to priority.

Agree. Eg. Ethereum is completely pointless if dapp developers don't make something useful of it.

**Comment by lrettig** [comment]
It's about timing, and again, about controversiality. I agree that, without apps, the network is pretty useless. But we need a stable base layer before those apps can thrive. And, while funding this or that dapp project could be controversial, funding one of the mainnet clients that's running 40 or 50% of the network should be a lot less controversial. Let's be incremental here, fund the uncontroversial stuff first, and not bite off more than we can chew.

**Comment by pet3r-pan** [comment]
> It's about timing, and again, about controversiality. I agree that, without apps, the network is pretty useless. But we need a stable base layer before those apps can thrive. And, while funding this or that dapp project could be controversial, funding one of the mainnet clients that's running 40 or 50% of the network should be a lot less controversial. Let's be incremental here, fund the uncontroversial stuff first, and not bite off more than we can chew.

Imo, dapps rn are beginning to thrive today. But point taken. Agree with limiting the controversial stuff in the beginning and going for the easy wins first.
